src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -2,15 +2,19 @@
 
 public class AuthenticationService {
     /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+       * Imagine that some user wants to login to your site.
+       * You should check if user credentials (login and password) are valid or not.
+       * All users are stored in <code>UserService</code> class.
+       *
+       * @param email    - user's email
+       * @param password - user's password
+       * @return true if user by email exists and passed password is equal to user's password.
+       * Return false in any other cases.
+       */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        return userService.findByEmail(email) != null
+                    && userService.findByEmail(email).getPassword().equals(password);
```

Comment:
Better create new variable - user
```suggestion
        User user = userService.findByEmail(email);
        return user != null && user.getPassword().equals(password);
```